modules/local/consolidatebcfs/main.nf

```diff
@@ -17,15 +17,16 @@ process CONSOLIDATE_BCFS {
 
     output:
     tuple val(meta), path( "${meta.id}_consolidated.bcf" ), emit: consolidated_bcfs
-    path( "${meta.id}_consolidated.vcf" )                 , emit: consolidated_vcfs
```

Comment:
This was removed because it wasn't a VCF (it was stdout) and it wasn't used later in the workflow.

Reply:
Thanks so much.